modAionImpl/src/org/aion/zero/impl/config/dynamic/InFlightConfigReceiver.java

```diff
@@ -110,12 +110,8 @@ ConfigProposalResult applyNewConfig(Cfg newCfg) throws RollbackException {
                         String.format(
                                 "The key '%s' does not support in-flight configuration change",
                                 key));
-            } else if (maybeApplier == null) {
```

Comment:
Shouldn't the null check come before the `isPresent()` call? If `maybeApplier` is null line 108 will throw an exception.

Reply:
cause `if (!maybeApplier.isPresent())` already checked the nullity (it's an Optional object).

Reply:
sorry, misread the change as an addition, not removal